src/store/reducers/appReducer.js

```diff
@@ -129,9 +161,14 @@ const initialState = {
   logo: svgs.contoso,
   name: 'companyName',
   isDefaultLogo: true,
-  azureMapsKey: '',
   deviceGroupFlyoutIsOpen: false,
   timeInterval: 'PT1H',
+  settings: {
```

Comment:
Make a const for these defaults in the service model... see that file for more info.

Reply:
which service model? Also I'm not convinced on why we have to do that.